aws-lc-rs/src/lib.rs

```diff
@@ -128,6 +128,7 @@ mod fips;
 pub mod iv;
 mod ptr;
 mod rsa;
+pub mod tlsprf;
```

Comment:
Minor nit: Should we do `tls_prf` or `tlsprf`?
```suggestion
pub mod tls_prf; 
```